index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Assignment-2</title>
+</head>
+<body>
+<!-- header start -->
+    <header>
+        <div class="logo"><h2>Team Shivam</h2></div>
```

Comment:
provide a image logo 

`<img src="https://www.seekpng.com/png/detail/77-772362_youtube-logo-youtube-logo-png.png">` according to design we need to provide image logo